Detections/Box/BoxInactiveUserLogin.yaml

```diff
@@ -0,0 +1,37 @@
+id: edbf38d7-e170-4af2-ad50-1a05b374611b
+name: Box - Inactive user login
+description: |
+  'Detects user login after long inactivity period.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: BoxDataConnector
+    dataTypes:
+      - BoxEvents_CL
+queryFrequency: 1h
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - InitialAccess
+query: |
+  let lbperiod_start = 90d;

```

Comment:
14d max lookback recommended, depending on the size of the data, this query lookback could cause performance issues for the customer who implement it.

Reply:
updated